backend/fms_core/services/sample.py

```diff
@@ -194,7 +195,8 @@ def transfer_sample(process: Process,
                 coordinates=coordinates_destination if coordinates_destination else "",
                 creation_date=execution_date,
                 volume=volume_destination if volume_destination else volume_used,
-                depleted=False
+                depleted=False,
+                comment=comment if comment else {},
```

Comment:
This will override the sample based comment instead of setting it on the process_measurement. I do not think we should do that. Also, you should not put a dictionnary ({}) in a string variable. Use None or "".